build.sh

```diff
@@ -137,7 +149,11 @@ if [ "${local}" -eq 1 ]; then
 fi
 
 # Use latest released Debian as our base image
-export DEBIAN_BASE_IMAGE="gcr.io/google-appengine/debian8:latest"
+if [ "${os_base}" == "ubuntu16" ]; then
+  export DEBIAN_BASE_IMAGE="gcr.io/gcp-runtimes/ubuntu_16_0_4:latest"
```

Comment:
Maybe change the variable name to `BASE_IMAGE` as it could be either debian or ubuntu?

Reply:
Renamed to OS_BASE_IMAGE